project/assets/scripts/controller/new-evaluation/new-evaluation.js

```diff
@@ -1,68 +1,78 @@
-const NewEvaluationEvents = function () {
-    this.init = function (redirectToFunction) {
-        setNavEvent('evaluations', addEvent.bind(null, 'evaluations', redirectToFunction));
-        document.getElementById('evaluations').classList.remove('navigation-bar-active');
-        document.getElementById('new-evaluation').classList.add('navigation-bar-active');
-        setNavEvent('logout', addEvent.bind(null, 'login', redirectToFunction));
+(function () {
+    const NewEvaluationEvents = function () {
+        this.init = function (redirectToFunction) {
+            setNavEvent('evaluations', addEvent.bind(null, 'evaluations', redirectToFunction));
+            document.getElementById('evaluations').classList.remove('navigation-bar-active');
+            document.getElementById('new-evaluation').classList.add('navigation-bar-active');
+            setNavEvent('logout', addEvent.bind(null, 'login', redirectToFunction));
 
-        const submitBtn = document.getElementById('submit-button');
-        submitBtn.addEventListener('click', getFormData);
+            const submitBtn = document.getElementById('submit-button');
+            submitBtn.addEventListener('click', getFormData);
+        };
+        this.destroy = function (redirectToFunction) {
+            removeEvents('evaluations', addEvent.bind(null, 'evaluations', redirectToFunction));
+            removeEvents('logout', addEvent.bind(null, 'login', redirectToFunction));
+            removeEvents('submit-button', getFormData);
+        };
     };
-    this.destroy = function (redirectToFunction) {
-        removeEvents('evaluations', addEvent.bind(null, 'evaluations', redirectToFunction));
-        removeEvents('logout', addEvent.bind(null, 'login', redirectToFunction));
-        removeEvents('submit-button', getFormData);
-    };
-};
 
-function getSelected(selectId) {
-    const e = document.getElementById(`${selectId}`);
-    return e.options[e.selectedIndex].value;
-}
+    function getSelected(selectId) {
+        const e = document.getElementById(`${selectId}`);
```

Comment:
What does `${selectId}` evaluate to, though?

Reply:
I've set the IDs of select tags to 'legend' + 'label' (ex. OOPDesignPatternsClasses)

Reply:
Fair. What I was getting at is that `${selectId}` is most likely synonymous to `selectId`, assuming that `selectId` is a string to start with.